plugins/oauth/hooks.js

```diff
@@ -74,7 +74,13 @@ module.exports = Object.assign({}, StoreMixin, {
   },
 
   redirectToLogin(nextState, replace) {
```

Comment:
Do we really need the whole new state? Please try to only pass arguments and properties that you actually need, as it's otherwise really hard to maintain the respective function/component.